testcases/OpTestLPM.py

```diff
@@ -59,3 +95,7 @@ def lpar_migrate_test(self):
 
     def runTest(self):
         self.lpar_migrate_test()
+
+    def tearDown(self):
+        if self.firewall_status:
+            self.cv_HOST.host_run_command("systemctl start firewalld.service")
```

Comment:
Does this gets called in exception path as well?

Reply:
Yes, it will be called. I verified it.